components/x-article-save-button/stories/index.js

```diff
@@ -0,0 +1,12 @@
+const { ArticleSaveButton } = require('../');
+
+exports.component = ArticleSaveButton;
+exports.package = require('../package.json');
+exports.knobs = require('./knobs');
+exports.dependencies = {
+	'o-normalise': '^1.6.0',
+	'o-typography': '^5.5.0'
```

Comment:
Do we need to include `o-icons` and the ones listed here? https://github.com/Financial-Times/x-dash/pull/164/files#diff-ef020e752ebe496f72a896942f3824ccR1